backend/entityservice/object_store.py

```diff
@@ -30,7 +30,7 @@ def connect_to_upload_object_store():
         config.UPLOAD_OBJECT_STORE_ACCESS_KEY,
         config.UPLOAD_OBJECT_STORE_SECRET_KEY,
         region="us-east-1",
```

Comment:
why is region hard-coded when everything else is configurable?

Reply:
It could be configurable but we'd probably have to introduce another minio/boto client just to connect to `us-east-1` for the granting the temporary credentials.